lib/stdlib/test/run_pcre_tests.erl

```diff
@@ -1063,6 +1063,21 @@ dumponesplit(F,{RE,Line,O,TS}) ->
 			    dsafe(safe(Str)),
 			    dsafe2(safe(RE)),
 			    NO++[{parts,2}]]),
+
+		 %% Use a special separator here ('§') to avoid conflicts
+		 io:format(F,"perl -e '$x = join(\"§\",split(/~s/~s,\"~s\",-1)); "
+			   "$x =~~ s/(^§+|§+$)//g; $x =~~ s/§§+/§/g; $x =~~ s/§/:/g; "
+			   "$x =~~ s/\\\\/\\\\\\\\/g; $x =~~ s/\\\"/\\\\\"/g; "
+			   "print \"    <<\\\"$x\\\">> = "
+			   "iolist_to_binary(join(re:split(\\\"~s\\\","
+			   "\\\"~s\\\",~p))), \\n\";'~n",
```

Comment:
Remove the space before "\\n" (I know that the other io:formats have space in that position, but there is no need to add redundant spaces to new lines).